src/main/java/com/github/instagram4j/instagram4j/responses/highlights/HighlightsMediaResponse.java

```diff
@@ -0,0 +1,16 @@
+package com.github.instagram4j.instagram4j.responses.highlights;
```

Comment:
Is it possible to use this response for `FeedReelsMediaRequest` as well?

If so, all is needed is to change the response for `FeedReelsMediaRequest` to use this response. Also following convention the response should be named `FeedReelsMediaResponse`.

The HighlightsMediaRequest is similar to FeedReelsMediaRequest so it's not needed to add this redundant request and should be removed.

Reply:
Modified the files as per standard of Instagram4j. Now response object is `FeedReelsMediaResponse` .